ab_tool/models.py

```diff
@@ -119,9 +119,9 @@ def to_json(self):
                         # databaseName stores the track name in the database for collision comparison
                         "databaseName": t.name,
                         # newName and editing are used to hold temporary values for track name editing
-                        "newName" : t.name,
+                        "newName": t.name,
                         "editing": False,
-                        "deleteURL": reverse('ab_testing_tool_delete_track', args=(t.id,))}
+                        "deleteURL": reverse('ab_testing_tool_delete_track', args=(resource_link_id, t.id,))}
```

Comment:
There's a bunch more of these throughout the code, fwiw.  Not a big deal, obviously, worth cleaning up as a followon change (i'll do it) if this lands at some point.